etl/steps/data/garden/AI/2023-06-07/monmoth_poll.py

```diff
@@ -0,0 +1,47 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from typing import cast
+
+import pandas as pd
+from owid.catalog import Dataset, Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("monmoth_poll.start")
+
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = cast(Dataset, paths.load_dependency("monmoth_poll"))
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["monmoth_poll"]
+    df = pd.DataFrame(tb)
```

Comment:
We should try to always work with tables, unless it's strictly necessary to work with pandas dataframes. But for this PR you can leave it like this.